Library/Homebrew/utils.rb

```diff
@@ -7,6 +7,9 @@
 require "utils/git"
 require "open-uri"
 
+HOMEBREW_UNICODE_TICK = ["2714".hex].pack("U*")
+HOMEBREW_UNICODE_CROSS = ["2718".hex].pack("U*")
```

Comment:
Could the `Tty` class be a better place to host these constants? We could remove the `HOMEBREW_` prefix then.